.requirements-pyside.txt

```diff
@@ -0,0 +1,4 @@
+# this is a seperate file, referenced in requirements-optional.txt,
+# because pip does not accept the necessary options per-requirement
+--only-binary --no-index --find-links https://pymor.github.io/wheels/
```

Comment:
Unfortunately, the pyside wheel for Py34 does not work on my machine. I am getting
```
ImportError: libpyside.cpython-34m.so.1.2: cannot open shared object file: No such file or directory
```
I guess this has something to do with my virtualenv setup. Could you check if the wheels work for you in a virtualenv?